job-dsls/jobs/kogito.groovy

```diff
@@ -303,6 +308,60 @@ job("${folderPath}/kogito-cloud-${mainBranch}") {
 
 }
 
+
+// ++++++++++++++++++++++++++++++++++++++++++ Build and deploys kogito-cloud-s2i-images ++++++++++++++++++++++++++++++++++++++++++++++++++
+
+// definition of kogito-cloud-s2i-images script
+def kogitoCloudS2i='''#!/bin/bash -e
+
+# clone the kogito-bom repository
+git clone https://github.com/$ghOrgUnit/kogito-cloud.git -b $mainBranch --depth 50
```

Comment:
Please consider moving the `git clone` command from shell step to a scm part of the job config - not needed to be done in this PR, it is used this way in other places as well. It would use a Jenkins plugin for cloning which is a best practice.

Reply:
How and where can I do this?

Reply:
@spolti An example can be found here: https://github.com/kiegroup/kie-jenkins-scripts/blob/94368875384b24145d8681fed285f1c5586cbd6d/job-dsls/jobs/kie_dailyBuild_pipeline.groovy#L376

But I would suggest making that change in another PR, because it should affect all kogito jobs ideally, so the definitions are consistent.

Reply:
it is just a copy for the kogito-cloud, shouldn't that job do it as well?

Reply:
all jobs on that scripts does exactly the same thing, maybe a subsequent jira to address all other jobs will be necessary.